src/functional/swing_slide_filter.zig

```diff
@@ -290,10 +278,10 @@ fn appendValue(comptime T: type, value: T, compressed_values: *std.ArrayList(u8)
     }
 }
 
-/// Computes the intercept coefficient of a linear function that passes through the point
-/// (time, value) and with the given slope coefficient.
-fn computeInterceptCoefficient(slope: f80, time: usize, value: f64) f80 {
-    return value - slope * usizeToF80(time);
+/// Computes the intercept coefficient of a linear function that passes through `point` and
+/// with the given slope coefficient.
```

Comment:
Very minor, but either with or and. Also slope lacks backticks.

Reply:
Fixed.